sotodlib/site_pipeline/make_book.py

```diff
@@ -67,11 +104,14 @@ def get_parser(parser=None):
         type=str, 
         help="Path to imprinter configuration file"
     )
-    parser.add_argument('output_root', type=str, help="Root path of the books")
+    #parser.add_argument('output_root', type=str, help="Root path of the books")
+    parser.add_argument("--parallel-oper", action="store_true")
     return parser
 
 
 if __name__ == "__main__":
+    multiprocessing.set_start_method('spawn')
```

Comment:
So this doesn't need to be in main()?

Reply:
You're right, I probably do need to put it in `main()` I haven't tested this through prefect yet but I don't think this section gets executed by prefect.